scripts/install_vulkan_headers.sh

```diff
@@ -1,8 +1,54 @@
 #!/bin/bash
 
-git clone https://github.com/KhronosGroup/Vulkan-Headers.git "${temp_dir}/Vulkan-Headers"
+#--- Script for installing Vulkan Headers ---#
+
+
+### Setup ###
+
+## Source the default values for imported variables
+. scripts/default_values.sh
+
+## Source the global helper functions
+. scripts/system_utils.sh
+
+## Source the configurations for our dependencies
+. deps.sh
+
+
+### Package metadata setup ###
+
+if [ -z "${dep_name}" ]; then
+    dep_name="${dep_name_vulkan}"
+fi
+
+if [ -z "${src_dir}" ]; then
+    src_dir="${parent_dir_vulkan}/${dep_name_vulkan}"
+fi
+
+if [ -z "${dep_ver}" ]; then
+    dep_ver="${dep_ver_vulkan}"
+fi
+
+repo_url="https://github.com/KhronosGroup/Vulkan-Headers.git"
+build_dir="${src_dir}/build"
+#prefix_dir="install" # Override default
+
+
+### Fetch, build and install ###
+
+## Fetch
+git clone --depth 1 --branch "${dep_ver}" "${repo_url}" "${src_dir}"
+
+## Build
 cmake \
-	-S "${temp_dir}/Vulkan-Headers" \
-	-B "${temp_dir}/Vulkan-Headers/build" \
-	-D CMAKE_INSTALL_PREFIX=install
```

Comment:
Wait. Where was this being installed previously?

Reply:
To the `install` directory at the root of the ILLIXR project. Not sure why this was chosen, but hopefully changing it to the usual `/usr/local` is safe.